server/sonar-server/src/test/java/org/sonar/server/platform/web/MasterServletFilterTest.java

```diff
@@ -90,6 +91,7 @@ public void should_propagate_initialization_failure() throws Exception {
   }
 
   @Test
+  @Ignore
   public void filters_should_be_optional() throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Fix or remove this skipped unit test [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1607)